From fdb3e6ddf5db1ac6c5d651e37bbbb27d7a1ab75f Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Tue, 24 Aug 2021 19:14:50 +0100
Subject: [PATCH 1/2] Set CONFIG_INSTALL_DIR to share/eigenpy/cmake

---
 cmake/package-config.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/package-config.cmake b/cmake/package-config.cmake
index 7d8e8da..90e25d7 100644
--- a/cmake/package-config.cmake
+++ b/cmake/package-config.cmake
@@ -28,7 +28,7 @@ macro(_SETUP_PROJECT_PACKAGE_INIT)
 
   # Layout. This works for all platforms: * <prefix>/lib/cmake/<PROJECT-NAME> *
   # <prefix>/lib/ * <prefix>/include/
-  set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
+  set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/cmake")
   set(INCLUDE_INSTALL_DIR "include")
   set(INCLUDE_INSTALL_DESTINATION "${INCLUDE_INSTALL_DIR}/${PROJECT_NAME}")
 
@@ -146,7 +146,7 @@ macro(SETUP_PROJECT_PACKAGE_FINALIZE)
 
   # Layout. This works for all platforms: * <prefix>/lib/cmake/<PROJECT-NAME> *
   # <prefix>/lib/ * <prefix>/include/
-  set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
+  set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/cmake")
   set(INCLUDE_INSTALL_DIR "include")
   set(INCLUDE_INSTALL_DESTINATION "${INCLUDE_INSTALL_DIR}/${PROJECT_NAME}")
 
-- 
2.25.1

