From b91744b902ef029963357c178d572aebb3709ab3 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Fri, 24 Apr 2020 15:29:32 +0100
Subject: [PATCH 2/6] Explicitly set Python3 for Debian Buster

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4d41ca6..7cd02ab 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,6 +32,8 @@ if(NOT EXISTS "${CMAKE_SOURCE_DIR}/cmake/base.cmake")
 endif()
 
 set(CMAKE_BUILD_TYPE "Release")
+set(PYTHON_EXECUTABLE "python3") # specify python3 so that Debian Buster passes
+
 
 # Disable -Werror on Unix for now.
 set(CXX_DISABLE_WERROR True)
-- 
2.25.1

