From 311c84d15cb64726750d19a49ef7f2b27dfad14b Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Fri, 24 Apr 2020 15:29:32 +0100
Subject: [PATCH 2/3] Explicitly set Python3 for Debian Buster

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ab7fef9..2c89bf9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,6 +36,9 @@ SET(CMAKE_VERBOSE_MAKEFILE True)
 if(NOT DEFINED CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
+if(NOT DEFINED PYTHON_EXECUTABLE)
+  set(PYTHON_EXECUTABLE "python3") # specify python3 so that Debian Buster passes
+endif()
 
 # ----------------------------------------------------
 # --- OPTIONS  ---------------------------------------
-- 
2.25.1

