From 488fbee796c2387897481545d1a051b2171bea14 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Tue, 24 Aug 2021 19:17:07 +0100
Subject: [PATCH 2/2] Only set CMake variables if not set, cf. #2

---
 CMakeLists.txt | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index fa9e1e7..77ac556 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -43,6 +43,16 @@ else()
   endif()
 endif()
 
+IF(NOT DEFINED CMAKE_BUILD_TYPE)
+  SET(CMAKE_BUILD_TYPE "Release")
+ENDIF()
+IF(NOT DEFINED PYTHON_EXECUTABLE)
+  SET(PYTHON_EXECUTABLE "/usr/bin/python3") # specify python3 so that Debian Buster passes
+ENDIF()
+IF(NOT DEFINED CMAKE_INSTALL_LIBDIR)
+  SET(CMAKE_INSTALL_LIBDIR "lib/")
+ENDIF()
+
 function(set_standard_output_directory target)
   set_target_properties(
     ${target}
-- 
2.25.1

