From 51dc949ce82950f9f28578ace35879d04bd5fed4 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Tue, 24 Aug 2021 19:08:40 +0100
Subject: [PATCH 2/2] Set CONFIG_INSTALL_DIR to share/eigenpy/cmake

---
 cmake/package-config.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/package-config.cmake b/cmake/package-config.cmake
index e95b2b0..5b226b2 100644
--- a/cmake/package-config.cmake
+++ b/cmake/package-config.cmake
@@ -28,7 +28,7 @@ macro(_SETUP_PROJECT_PACKAGE_INIT)
 
   # Layout. This works for all platforms: * <prefix>/lib/cmake/<PROJECT-NAME> *
   # <prefix>/lib/ * <prefix>/include/
-  set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
+  set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/cmake")
   set(INCLUDE_INSTALL_DIR "include")
   set(INCLUDE_INSTALL_DESTINATION "${INCLUDE_INSTALL_DIR}/${PROJECT_NAME}")
 
@@ -143,7 +143,7 @@ macro(SETUP_PROJECT_PACKAGE_FINALIZE)
 
   # Layout. This works for all platforms: * <prefix>/lib/cmake/<PROJECT-NAME> *
   # <prefix>/lib/ * <prefix>/include/
-  set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
+  set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/cmake")
   set(INCLUDE_INSTALL_DIR "include")
   set(INCLUDE_INSTALL_DESTINATION "${INCLUDE_INSTALL_DIR}/${PROJECT_NAME}")
 
-- 
2.25.1

