From c784b97dd85f28c491130617babeefe24e630b1d Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Fri, 24 Apr 2020 15:29:32 +0100
Subject: [PATCH 2/5] Explicitly set Python3 for Debian Buster

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b619ec6..6b79741 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,6 +39,9 @@ set(CMAKE_VERBOSE_MAKEFILE True)
 if(NOT DEFINED CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
+if(NOT DEFINED PYTHON_EXECUTABLE)
+  set(PYTHON_EXECUTABLE "python3") # specify python3 so that Debian Buster passes
+endif()
 
 # ----------------------------------------------------
 # --- OPTIONS  ---------------------------------------
-- 
2.25.1

