From 76f8f07b3c1368687f6ccdadb20a2b4e2c8f535d Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Tue, 7 Apr 2020 10:50:31 +0100
Subject: [PATCH 2/3] Patch: Install config to share/cmake

---
 cmake/package-config.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/package-config.cmake b/cmake/package-config.cmake
index fd9e8e0..d9be32f 100644
--- a/cmake/package-config.cmake
+++ b/cmake/package-config.cmake
@@ -34,7 +34,7 @@ MACRO(_SETUP_PROJECT_PACKAGE_INIT)
 #   * <prefix>/lib/cmake/<PROJECT-NAME>
 #   * <prefix>/lib/
 #   * <prefix>/include/
-set(CONFIG_INSTALL_DIR "lib/cmake/${PROJECT_NAME}")
+set(CONFIG_INSTALL_DIR "share/cmake/${PROJECT_NAME}")
 set(INCLUDE_INSTALL_DIR "include")
 set(INCLUDE_INSTALL_DESTINATION "${INCLUDE_INSTALL_DIR}/${PROJECT_NAME}")
 
-- 
2.25.1

