From 0b708a86f386c3e29f5583c68a5bf2e61718b3f9 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Fri, 24 Apr 2020 15:29:32 +0100
Subject: [PATCH 2/6] Explicitly set Python3 for Debian Buster

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index df0bf9e..1548dfc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,6 +29,8 @@ IF(NOT EXISTS "${CMAKE_SOURCE_DIR}/cmake/base.cmake")
 ENDIF()
 
 set(CMAKE_BUILD_TYPE "Release")
+set(PYTHON_EXECUTABLE "python3") # specify python3 so that Debian Buster passes
+
 
 # Disable -Werror on Unix for now.
 SET(CXX_DISABLE_WERROR True)
-- 
2.25.1

