From 1a9b5826842e7c8b154a2fac052808d63fd4d766 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Thu, 9 Apr 2020 17:23:45 +0100
Subject: [PATCH 2/3] Set build type to Release

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a15345a..45c4323 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -21,6 +21,7 @@ ENDIF()
 SET(CXX_DISABLE_WERROR True)
 SET(CMAKE_VERBOSE_MAKEFILE True)
 
+set(CMAKE_BUILD_TYPE "Release")
 set(CMAKE_INSTALL_LIBDIR "lib/")
 
 INCLUDE(cmake/base.cmake)
-- 
2.17.1

