From ac706c0c9ad77865d1ebfc7dc15e24f29c60f9bc Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Thu, 9 Apr 2020 17:24:43 +0100
Subject: [PATCH 3/4] Set CONFIG_INSTALL_DIR to share/cmake/eigenpy

---
 cmake/package-config.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/package-config.cmake b/cmake/package-config.cmake
index bf000f1..cf7606f 100644
--- a/cmake/package-config.cmake
+++ b/cmake/package-config.cmake
@@ -34,7 +34,7 @@ MACRO(_SETUP_PROJECT_PACKAGE_INIT)
 #   * <prefix>/lib/cmake/<PROJECT-NAME>
 #   * <prefix>/lib/
 #   * <prefix>/include/
-set(CONFIG_INSTALL_DIR "lib/cmake/${PROJECT_NAME}")
+set(CONFIG_INSTALL_DIR "share/cmake/${PROJECT_NAME}")
 set(INCLUDE_INSTALL_DIR "include")
 set(INCLUDE_INSTALL_DESTINATION "${INCLUDE_INSTALL_DIR}/${PROJECT_NAME}")
 
@@ -106,7 +106,7 @@ MACRO(SETUP_PROJECT_PACKAGE_FINALIZE)
 #   * <prefix>/lib/cmake/<PROJECT-NAME>
 #   * <prefix>/lib/
 #   * <prefix>/include/
-set(CONFIG_INSTALL_DIR "lib/cmake/${PROJECT_NAME}")
+set(CONFIG_INSTALL_DIR "share/cmake/${PROJECT_NAME}")
 set(INCLUDE_INSTALL_DIR "include")
 set(INCLUDE_INSTALL_DESTINATION "${INCLUDE_INSTALL_DIR}/${PROJECT_NAME}")
 
-- 
2.17.1

