From fc406254716645d067b44e062e0b5491ef388768 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Fri, 29 May 2020 11:47:27 +0100
Subject: [PATCH 3/6] Workaround for architecture issue with GNUInstallDirs

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3d21dad..dcf8b3c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,7 +30,7 @@ ENDIF()
 
 set(CMAKE_BUILD_TYPE "Release")
 set(PYTHON_EXECUTABLE "python3") # specify python3 so that Debian Buster passes
-
+set(CMAKE_INSTALL_LIBDIR "lib/")
 
 # Disable -Werror on Unix for now.
 SET(CXX_DISABLE_WERROR True)
-- 
2.25.1

