From b3be4f2349f25f7fea0b6e90c57ab604d3a92d72 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Tue, 26 Jan 2021 23:32:03 +0000
Subject: [PATCH 4/6] Use full path to Python3, cf. #1

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5a14e14..ad95b4d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -28,7 +28,7 @@ IF(NOT EXISTS "${CMAKE_SOURCE_DIR}/cmake/base.cmake")
 ENDIF()
 
 set(CMAKE_BUILD_TYPE "Release")
-set(PYTHON_EXECUTABLE "python3") # specify python3 so that Debian Buster passes
+set(PYTHON_EXECUTABLE "/usr/bin/python3") # specify python3 so that Debian Buster passes
 set(CMAKE_INSTALL_LIBDIR "lib/")
 
 # Disable -Werror on Unix for now.
-- 
2.25.1

