From cbd4af836f102b23f6a5ac2e227d77c7f37fa7e6 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Tue, 26 Jan 2021 23:32:03 +0000
Subject: [PATCH 4/6] Use full path to Python3, cf. #1

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0ecf39a..f451952 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,7 +32,7 @@ if(NOT EXISTS "${CMAKE_SOURCE_DIR}/cmake/base.cmake")
 endif()
 
 set(CMAKE_BUILD_TYPE "Release")
-set(PYTHON_EXECUTABLE "python3") # specify python3 so that Debian Buster passes
+set(PYTHON_EXECUTABLE "/usr/bin/python3") # specify python3 so that Debian Buster passes
 set(CMAKE_INSTALL_LIBDIR "lib/")
 
 # Disable -Werror on Unix for now.
-- 
2.43.0

