From 4f13c6b0853b05a96131b6e2390a8431effab594 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Tue, 2 Aug 2022 14:45:24 +0100
Subject: [PATCH 4/5] ROS2: Set PYTHON_DEB_LAYOUT and PYTHON_STANDARD_LAYOUT

---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c40715f..b78fce9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -63,6 +63,10 @@ include("${JRL_CMAKE_MODULES}/base.cmake")
 compute_project_args(PROJECT_ARGS LANGUAGES CXX)
 project(${PROJECT_NAME} ${PROJECT_ARGS})
 
+# Required for ROS2
+set(PYTHON_DEB_LAYOUT OFF)
+set(PYTHON_STANDARD_LAYOUT ON)
+
 include("${JRL_CMAKE_MODULES}/boost.cmake")
 include("${JRL_CMAKE_MODULES}/python.cmake")
 include("${JRL_CMAKE_MODULES}/ide.cmake")
-- 
2.25.1

