From 897ca25f43414828df918f54db7c596ac9ad2d72 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Fri, 29 May 2020 11:47:27 +0100
Subject: [PATCH 5/5] Workaround for architecture issue with GNUInstallDirs

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5533f15..b93c9f4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -19,7 +19,7 @@ ENDIF()
 
 set(CMAKE_BUILD_TYPE "Release")
 set(PYTHON_EXECUTABLE "python3") # specify python3 so that Debian Buster passes
-
+set(CMAKE_INSTALL_LIBDIR "lib/")
 
 # Disable -Werror on Unix for now.
 SET(CXX_DISABLE_WERROR True)
-- 
2.17.1

