From 56e0c83ddf6c423918386156b4f8351e2f0bc180 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Tue, 24 Aug 2021 19:14:50 +0100
Subject: [PATCH 5/6] Set CONFIG_INSTALL_DIR to share/eigenpy/cmake

---
 cmake/package-config.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/package-config.cmake b/cmake/package-config.cmake
index d7134c1..4d6c320 100644
--- a/cmake/package-config.cmake
+++ b/cmake/package-config.cmake
@@ -28,7 +28,7 @@ macro(_SETUP_PROJECT_PACKAGE_INIT)
 
   # Layout. This works for all platforms: * <prefix>/lib/cmake/<PROJECT-NAME> *
   # <prefix>/lib/ * <prefix>/include/
-  set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
+  set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/cmake")
   set(INCLUDE_INSTALL_DIR "include")
   set(INCLUDE_INSTALL_DESTINATION "${INCLUDE_INSTALL_DIR}/${PROJECT_NAME}")
 
@@ -143,7 +143,7 @@ macro(SETUP_PROJECT_PACKAGE_FINALIZE)
 
   # Layout. This works for all platforms: * <prefix>/lib/cmake/<PROJECT-NAME> *
   # <prefix>/lib/ * <prefix>/include/
-  set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
+  set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/cmake")
   set(INCLUDE_INSTALL_DIR "include")
   set(INCLUDE_INSTALL_DESTINATION "${INCLUDE_INSTALL_DIR}/${PROJECT_NAME}")
 
-- 
2.43.0

